src/test/java/email/KafkaEmailClientUnitTest.java

```diff
@@ -0,0 +1,105 @@
+package email;
+
+import org.apache.avro.Schema;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.strikeoffobjections.common.FormatUtils;
+import uk.gov.companieshouse.api.strikeoffobjections.email.AvroSerializer;
+import uk.gov.companieshouse.api.strikeoffobjections.email.KafkaEmailClient;
+import uk.gov.companieshouse.api.strikeoffobjections.model.model.EmailContent;
+import uk.gov.companieshouse.api.strikeoffobjections.utils.Utils;
+import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
+import uk.gov.companieshouse.service.ServiceException;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.function.Supplier;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class KafkaEmailClientUnitTest {
+
+    private static final LocalDateTime CREATED_AT =
+            LocalDateTime.of(2019, 1, 1, 0, 0);
+    private static final Future<RecordMetadata> mockedFuture = Mockito.mock(Future.class);
+    private static final Future<RecordMetadata> faultyMmockedFuture = Mockito.mock(Future.class);
```

Comment:
private static final variables should be all caps, also this one has a typo

Reply:
They certainly should.